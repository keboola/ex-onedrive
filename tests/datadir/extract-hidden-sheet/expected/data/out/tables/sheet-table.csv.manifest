{"columns":["Col_4","Col_5","Col_6"]}
