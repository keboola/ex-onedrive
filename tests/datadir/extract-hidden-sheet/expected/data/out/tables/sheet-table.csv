"A","B","cell from hidden sheet"
"X","Y","Z"
